public/index.html

```diff
@@ -0,0 +1,120 @@
+            <ol>
+              <li>Go to <kbd class='cmp_input'>public/index.html</kbd> in your project folder and open it.</li>
+              <li>Uncomment line 11 to show the h1 tag, save the file, and refresh the app.</li>
+              <li>For more info, checkout out our <a target="_blank" rel="noreferrer"
+                  href="https://docs.architect.io/">docs</a>.</li>
+            </ol>
+            <br />
+            <li>Deploy your app to the cloud via Architect <b>(Recommended)</b></li>
+            <ol>
+              <li>Login or register a new Architect account for free at <a target="_blank" rel="noreferrer"
+                  href="https://cloud.architect.io/signup">architect.io</a></li>
+              <li>Authenticate using the CLI by running the command <kbd class="cmp_input">architect login</kbd></li>
+              <li>Deploy your app to the cloud using the command <kbd class="cmp_input">architect deploy</kbd></li>
+            </ol>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="container">
+      <div style="padding-bottom: 20px">
+        <div id="card-header">
+          <h4><b>Movie Ratings</b></h4>
+        </div>
+        <div class="user_inputs">
+          <label>
+            Movie Title
+          </label>
+          <input id="movie-title" placeholder="Movie Title" type="text" name="name" class="form-control" value=""
+            minlength="1" maxlength="80" required>
+        </div>
+        <div class="user_inputs">
+          <label>
+            Rating
+          </label>
+          <input id="movie-rating" placeholder="Rating 1-5" type="number" name="rating" class="form-control" value=""
+            min="1" max="5" required>
+        </div>
+        <div class="d-grid gap-2">
+          <button type="submit" onclick="postData()">
+            Submit
+          </button>
+        </div>
+      </div>
+
+      <div th:unless="${#lists.isEmpty(movieRatings)}">
+        <div id="movieRatings"></div>
+      </div>
+    </div>
+  </div>
+  <script type="text/javascript">
+    function getData() {
+      fetch("/api/items").then(response => response.json())
+        .then((result) => {
+          document.getElementById("movieRatings").innerHTML = buildItemList(result);
+        })
+        .catch(error => console.log('error:', error));
+    }
+
+    function postData() {
+      const name = document.getElementById("movie-title").value;
+      const rating = document.getElementById("movie-rating").value;
+      fetch("/api/items", {
+        body: JSON.stringify({
+          name: name,
+          rating: rating
+        }),
+        method: "post",
+        headers: {
+          "Content-Type": "application/json"
+        }
+      }).then((result) => {
+        getData();
+      }).catch(error => console.log('error:', error));
+
+      document.getElementById("movie-title").reset();
+      document.getElementById("movie-rating").reset();
```

Comment:
Looks like there's an error with these two lines. The fields don't get cleared when the user submits the form

![image](https://user-images.githubusercontent.com/8558567/217892189-ff78ebf7-abb6-4740-ba87-3a73d6807aa7.png)

Reply:
I see that this just got changed, though we probably don't want to reload the page entirely. Setting the values of the fields back to `''` might be more appropriate